packages/odyssey-react/src/components/PasswordInput/PasswordInput.module.scss

```diff
@@ -0,0 +1,16 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.affixFull {
```

Comment:
@edburyenegren-okta - this css isn't correct anymore. I'm hoping you can look at the deployed storybook and tell me how to get the css around the buttons correct? It looks right when there is just an icon, or text but when a button is the suffix something isn't right.

Reply:
AFAIK this should continue to live in TextInput, since we will need it for cases other than PasswordInput. That might be part of the problem.